guide.md

```diff
@@ -9045,9 +8981,9 @@ Here are the steps for BleachBit:
 
 ##### Force a Trim with Windows Optimize (for SSD drives):
 
-With this Native Windows 10 utility, you can just trigger a Trim on your SSD which should be more than enough to securely clean all deleted files that somehow would have escaped Trim when deleting them.
+With this Native Windows 10/11 utility, you can just trigger a Trim on your SSD which should be more than enough to securely clean all deleted files that somehow would have escaped Trim when deleting them.
 
-Just open Windows Explorer, Right Click on your System Drive and click Properties. Select Tools. Click Optimize and then Optimize again. You are done. That is probably enough in my opinion.
+Just open Windows Explorer, Right Click on your System Drive and click Properties. Select Tools. Click Optimize and Defragment. You are done. That is probably enough in my opinion.

```

Comment:
Why are we de-fragmenting the SSD?

Reply:
It seems to be they should have changed this to be more clear, because it makes no sense to defragment an SSD. Only HDDs benefit from this and they are slower as a result of how they're designed. We should at least point this out?